kyuubi-hive-beeline/pom.xml

```diff
@@ -46,6 +46,11 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.thrift</groupId>
+            <artifactId>libthrift</artifactId>
```

Comment:
Beeline currently does not have a shade, and the code also has some thrift-related classes. 
If the server returns the thrift exception class of the shade, beeline may not be able to handle it correctly.

org.apache.hive.beeline.BeeLine#handleSQLException

https://github.com/apache/hive/blob/81b394104b08efdc42d782d444462f4fdc12a8fd/beeline/src/java/org/apache/hive/beeline/BeeLine.java#L2179-L2197

Reply:
yea, strictly speaking, we must fork and get rid of vanilla beeline to fix it thoroughly

Reply:
It sounds like we have to be compatible with two thrift exceptions, one from hiveserver2 and one from kyuubi server.

Reply:
let's try this way, the method visibility is package level, we can overwrite it in sub-class that located at the same pacakge